authorization_services/topics/service-authorization-uma-account-my-resources.adoc

```diff
@@ -0,0 +1,43 @@
+[[_service_authorization_api_aapi]]
+= Managing Access to Users Resources
+
+Users can manage access to their resources using the {project_name} User Account Service.
+
+image:{project_images}/service/account-my-resource.png[alt="My Resources"]
+
+On the left side menu there is a `My Resources` option that leads to a page where users are able to:
+
+* Manage Permission Rquests that *Need my approval*
++
+In this section, there is a list of all permission request waiting for user's approval. These request are related
```

Comment:
This section contains a list of all permission requests awaiting approval. These requests are connected to the parties (users) requesting access to...